Update README to clarify some of the steps (#3)
2uinc/devops-test: README.md

# SRE Technical Test
Infrastructure is best represented as code and provisioning of resources should be automated as much as possible. 

Here at 2U, [Terraform](terraform.io) is our tool of choice. Below you will find 2 tasks that will help demonstrate your apptitude for these required tools

> The first task, under `Infrustructure`, write some simple `terraform`. If you have any questions regarding how to work within this space, please ask your interviewers.

Reviewer: Infrastructure spelling correction on ln 6

For the second task we will assess your ability to write scripts. Scripts help us accomplish our day-to-day tasks more quickly. Whether it's a one off script to help import, migrate, or update terraform state. Or a task to help developers bootstrap application builds within our ci/cd pipelines.

Finally, we will do a bit of sys admin work.  For this we leverage https://sadservers.com which provids some scenarios to tackle.

## Setup
Please goto this url prior to the interview to ensure your workstation is able to access it and so you can get a feel for the vscode workspace. 
* https://gitpod.io/#https://github.com/2uinc/devops-test

---

## Infrastructure

1. Build out some Infrastructure for AWS using Terraform that can be used in a repeatable way. 

What to build:
    * Total of three (3) EC2 Instances with the following:
        * Instance Type: `m5.large`
        * AMI: `ami-094125af156557ca2`
        * Region: `us-west-2`
    * Each instance should have it's own 100G EBS Volume attached

>     * Each instance should have a separate `Name` tags in AWS

Reviewer 2: ```suggestion
    * Each instance should be tagged with a unique `Name`
```
